nucleus/annotation.py

```diff
@@ -121,6 +123,9 @@ class BoxAnnotation(Annotation):  # pylint: disable=R0902
             attach to this annotation.  Strings, floats and ints are supported best
             by querying and insights features within Nucleus. For more details see
             our `metadata guide <https://nucleus.scale.com/docs/upload-metadata>`_.
+        embedding_vector (Optional[List]): Custom embedding vector for this object annotation.
```

Comment:
Don't need types here, I think sphinx inserts them from the type annotation in the function signature.